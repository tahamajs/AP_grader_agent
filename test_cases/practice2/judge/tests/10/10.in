show_table_info 6
enter 30
reserve_table 30 6
show_table_info 6
reserve_table 9
reserve_table 26 2
reserve_table 2
show_table_info 1
reserve_table 29 5
show_table_info 5
switch 30
show_table_info 5
show_table_info 6
switch 29
show_table_info 5
show_table_info 6
exit 30
show_table_info 6
reserve_table 27 3
reserve_table 32 2
reserve_table 5 2
enter 21
reserve_table 21 2
reserve_table 28 4
reserve_table 3 3
reserve_table 7
reserve_table 15
reserve_table 16
reserve_table 10 4
exit 28
reserve_table 17 2
reserve_table 50 1
reserve_table 1 3
reserve_table 41 5
show_table_info 1
exit 16
show_table_info 1
reserve_table 45 6
reserve_table 49 2
reserve_table 14 1
exit 45
reserve_table 47
reserve_table 46
reserve_table 18
reserve_table 19
switch 50
reserve_table 8
reserve_table 20
reserve_table 38 4
exit 41
reserve_table 11
reserve_table 23
reserve_table 39 5
reserve_table 36 1
reserve_table 25 5
exit 9
exit 26
reserve_table 41 6
show_table_info 1
show_table_info 5
switch 25
show_table_info 1
show_table_info 5
