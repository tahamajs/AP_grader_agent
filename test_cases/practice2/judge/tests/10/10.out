Table ID: 6
People at the table:
Table remaining capacity: 3
Waiting queue length: 0
Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Mike sits at table 6
Table ID: 6
People at the table: Mike
Table remaining capacity: 2
Waiting queue length: 0
Sadegh sits at table 1
Walter sits at table 2
Amir Hossein sits at table 1
Table ID: 1
People at the table: Amir Hossein, Sadegh
Table remaining capacity: 2
Waiting queue length: 0
Gustavo sits at table 5
Table ID: 5
People at the table: Gustavo
Table remaining capacity: 2
Waiting queue length: 0
Mike switches seats with Gustavo!
Table ID: 5
People at the table: Mike
Table remaining capacity: 2
Waiting queue length: 0
Table ID: 6
People at the table: Gustavo
Table remaining capacity: 2
Waiting queue length: 0
Gustavo switches seats with Mike!
Table ID: 5
People at the table: Gustavo
Table remaining capacity: 2
Waiting queue length: 0
Table ID: 6
People at the table: Mike
Table remaining capacity: 2
Waiting queue length: 0
Mike exits!
Table ID: 6
People at the table:
Table remaining capacity: 3
Waiting queue length: 0
Jesse sits at table 3
Hasti sits at table 2
Fereshte sits at table 2
Table 1: 2 0
Table 2: 1 0
Table 5: 2 0
Table 6: 3 0
Table 3: 1 0
Table 4: 2 0
Tommy sits at table 2
Hank sits at table 4
Parsa sits at table 3
Mahdi enters the waiting queue of table 2
Erfan enters the waiting queue of table 2
Alex sits at table 1
Hamid sits at table 4
Hank exits!
Sipamo enters the waiting queue of table 2
Sobhan sits at table 1
Paria enters the waiting queue of table 3
Melika sits at table 5
Table ID: 1
People at the table: Alex, Amir Hossein, Sadegh, Sobhan
Table remaining capacity: 0
Waiting queue length: 0
Alex exits!
Table ID: 1
People at the table: Amir Hossein, Sadegh, Sobhan
Table remaining capacity: 1
Waiting queue length: 0
Marry sits at table 6
Parham enters the waiting queue of table 2
Babak sits at table 1
Marry exits!
Peyman enters the waiting queue of table 1
Shahin enters the waiting queue of table 2
Amin enters the waiting queue of table 1
Sina enters the waiting queue of table 1
Sobhan switches seats with Fereshte!
Taha enters the waiting queue of table 1
Reza enters the waiting queue of table 1
Aryan sits at table 4
Melika exits!
Shaghayegh enters the waiting queue of table 1
Narges enters the waiting queue of table 1
Asghar sits at table 5
Saeed enters the waiting queue of table 1
Saman sits at table 5
Sadegh exits!
Walter exits!
Melika sits at table 6
Table ID: 1
People at the table: Amir Hossein, Babak, Fereshte, Peyman
Table remaining capacity: 0
Waiting queue length: 7
Table ID: 5
People at the table: Asghar, Gustavo, Saman
Table remaining capacity: 0
Waiting queue length: 0
Saman switches seats with Amir Hossein!
Table ID: 1
People at the table: Babak, Fereshte, Peyman, Saman
Table remaining capacity: 0
Waiting queue length: 7
Table ID: 5
People at the table: Amir Hossein, Asghar, Gustavo
Table remaining capacity: 0
Waiting queue length: 0
